wrapper-kernel/jupyter_gap_wrapper/gap/setup.g

```diff
@@ -66,10 +66,58 @@ HELP_VIEWER_INFO.jupyter_online :=
              for r in GAPInfo.RootPaths do
                  p := ReplacedString(url, r, "https://cloud.gap-system.org/");
              od;
-             Print("<a target=\"_blank\" href=\"", p, "\">Help</a>\n");
+             Print("<a target=\"_blank\" href=\"", p, "\">click here</a><br/>\n");
          end
         );
 
+DeclareGlobalFunction("HELP_PRINT_URL");
+
+#############################################################################
+##
+#F  HELP_PRINT_URL( <match> ) . . . . . .  print the url for the help section
+##
+##  Based on HELP_PRINT_MATCH
+##
+##  <match> is [book, entrynr]
+##
+InstallGlobalFunction(HELP_PRINT_URL, function(match)
+local book, entrynr, viewer, hv, pos, type, data;
+  book := HELP_BOOK_INFO(match[1]);
+  entrynr := match[2];
+  viewer:= UserPreference("HelpViewers");
+  if HELP_LAST.NEXT_VIEWER = false then
+    hv := viewer;
+  else
+    pos := Position( viewer, HELP_LAST.VIEWER );
+    if pos = fail then
+      hv := viewer;
+    else
+      hv := viewer{Concatenation([pos+1..Length(viewer)],[1..pos])};
+    fi;
+    HELP_LAST.NEXT_VIEWER := false;
+  fi;
+  for viewer in hv do
+    # type of data we need now depends on help viewer
+    type := HELP_VIEWER_INFO.(viewer).type;
+    # get the data via appropriate handler
+    data := HELP_BOOK_HANDLER.(book.handler).HelpData(book, entrynr, type);
+    if data <> fail then
+      # show the data
+      HELP_VIEWER_INFO.(viewer).show(data);
```

Comment:
It is this line which calls `HELP_VIEWER_INFO.browser` and causes opening of all tabs. I suggest actually to not to use `HELP_VIEWER_INFO.browser` at all. We should have `HELP_VIEWER_INFO.jupyter_online` and its counterpart `HELP_VIEWER_INFO.jupyter_local` which does not translate URLs to point to the server hosting the notebook.